scene/gui/tree.cpp

```diff
@@ -3233,8 +3233,8 @@ void Tree::_go_up() {
 		if (!prev) {
 			return;
 		}
-		selected_item = prev;
-		emit_signal(SNAME("cell_selected"));
+
+		select_single_item(prev, get_root(), selected_col);
```

Comment:
Why is `select_single_item` used for `SELECT_MULTI`?

Reply:
Arrow key overrides it. It's exactly what happens in other systems, such as in Windows file explorer.

Reply:
Can you take a look at #92959? The code currently crashes, and when I fixed the crash, @KoBeWi reported incorrect behavior, that it selects items which are not visible.